babybuddy/settings/base.py

```diff
@@ -143,6 +146,12 @@
 
 LOGOUT_REDIRECT_URL = "babybuddy:login"
 
+# Only include when REVERSE_PROXY_AUTH env is 1
```

Comment:
I'm a bit confused here -- up at L67 this middleware is added to the end of the of `MIDDLEWARE` but not down here it's being added again? I assume we don't want the L67 change, right?

Reply:
You're correct, I'll be removing the L67 change.